Documentation/Configuration/FlagFiles/Index.rst

```diff
@@ -0,0 +1,108 @@
+.. include:: /Includes.rst.txt
+
+.. _flag-files:
+
+==========
+Flag files
+==========
+
+TYPO3 uses a set of special files known as flag files or indicator files to
+control and manage low-level configurations, behaviors, and security settings
+of the system. These files act as triggers that enable or disable specific
+features or functionalities in TYPO3, often without requiring direct
+modifications to the core configuration files.
+
+Flag files are typically placed in specific locations within the TYPO3 file
+system and are usually named in a way that reflects their purpose.
+
+Below is a list of commonly used TYPO3 flag files, along with explanations of
+their functions and typical use cases.
+
+..  confval-menu::
+    :display: table
+    :Location:
+    :Legacy Location:
+
+    ..  confval:: LOCK_BACKEND
+        :name: flag-file-lock-backend
+        :Content: empty or arbitrary text file
+        :Location: typo3conf/LOCK_BACKEND
+        :Legacy Location: typo3conf/LOCK_BACKEND
+        :TYPO3 Commands: `vendor/bin/typo3 backend:lock`, `vendor/bin/typo3 backend:unlock`
+
+        If the file is set an error message is displayed when you try to log into
```

Comment:
```suggestion
        If the file exists, an error message is displayed when you try to log into
```

Reply:
The file can also contain a URL as file contents, in which case a redirect to that location is performed instead of displaying a message; adapt ":Content:" for this?

Reply:
I dont really get your comment. You have an example? Since which version was that possible?